mw-webapp/src/component/table/renderCellValue/helpers/updateMentorComment.ts

```diff
@@ -0,0 +1,16 @@
+import {MentorCommentDAL} from "src/dataAccessLogic/MentorCommentDAL";
+import {MentorComment} from "src/model/businessModel/MentorComment";
+
+/**
+ * Update text in MentorComments cell
```

Comment:
Unclear content, what is mentor cell, what happened if tomorrow I will change the name, how to track it here?